styles/globals.css

```diff
@@ -1,3 +1,14 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+:root {
+  --font-kode-mono: "Kode Mono";
+}
+
+@font-face {
+  font-family: "Kode Mono";
+  src: url("/fonts/KodeMono.woff2") format("woff supports variations"),
+       url("/fonts/KodeMono.woff2") format("woff-variations");
+  font-weight: 400 700;
+}
```

Comment:
This here won't be necessary as we can use `localFont` from `next/font/local`

```suggestion

```

Reply:
awesome 👍